streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -752,6 +754,13 @@ public void shouldNotGetQueryMetadataWithPartitionerWhenNotRunningOrRebalancing(
         streams.queryMetadataForKey("store", "key", (topic, key, value, numPartitions) -> 0);
     }
 
+    @Test(expected = UnknownStateStoreException.class)
```

Comment:
I would prefer not to use this annotation but instead use `assertThrows()` in the test. I know that other tests also use the annotation but those are older tests, and we try to move off this old pattern in favor of using `assertThrows`.

Reply:
Got it, will do.